test/util/print.test.js

```diff
@@ -29,6 +34,16 @@ describe('print asciimath', function () {
   tests.forEach(t => testPrintStr(t[0], t[1]));
 });
 
+describe('print latex', function() {
+  const tests = [
+    ['2+3+4', '2+3+4'],
+    ['2 + (4 - x) + - 4', '2+\\left(4-x\\right)-4'],
```

Comment:
I discovered 'TestUtil.flattenAndParse()' and thought it would be good to use instead. I also found that many of the tests aren't using it. Is it ok if I switch them over to it?